src/heitech.zer0mqXt.core/Transport/Message.cs

```diff
@@ -73,10 +73,13 @@ internal static XtResult<TMessage> ParseRqRepMessage<TMessage>(this NetMQMessage
             bool isSuccess = Convert.ToBoolean(configuration.Serializer.Deserialize<string>(successFrame));
             if (!isSuccess)
             {
-                var exceptnText = configuration.Serializer.Deserialize<string>(payloadFrame);
-                var excption = new ZeroMqXtSocketException("Server failed with" + exceptnText);
-                configuration.Logger.Log(new DebugLogMsg(excption.Message));
-                return XtResult<TMessage>.Failed(excption, operation);
+                var exceptionText = configuration.Serializer.Deserialize<string>(payloadFrame);
```

Comment:
We're deserializing into `exceptionText` unconditionally, even if `DeveloperMode` is disabled.
A little optimization would be to only deserialize it needed when the `DeveloperMode` is enabled.

Reply:
Good point, I´ll fix it